week2/ShoppingList/cart.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Jun Pick List_Cart</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.2/css/all.min.css"> <!--폰트어썸 CDN방식으로 IMPORT-->
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <span id="top"></span> <!--맨 위로 돌아올 수 있도록-->
+  <header>
+    <a href="./index.html"><i class="fa-solid fa-house"></i></a>
+    <h1><a  href="#top">Jun Pick - Cart</a></h1>
+    <i id="threeBar" class="fa-solid fa-bars"></i>
+    
+  </header>
+  <main id="cartContainer">
+    <div class="tableContainer">
+      <table id="cartTable">
+        <thead class="theadFix">
+          <tr>
+            <th class="checkCol"><input id="allCheck" type="checkbox" ></th>
```

Comment:
여기 input태그가 안 닫혀있네요 !
```suggestion
            <th class="checkCol"><input id="allCheck" type="checkbox" /></th>
```

Reply:
오.. 감사합니다 ! 근데 저거 혹시 자동으로 닫는 플러그인이 있을까요?
prettier나 code formatter같은 것들 사용하긴 하는데, 항상 <input> </input> (단일 태그라서 <input/> 하나만 있어야 함에도 불구하고) 이런 식으로 닫혀서 일일히 제가 지워주어야 했는데 번거롭더라구요..

Reply:
Auto Rename Tag : 태그 이름을 수정할때, <>여는 태그와 </> 닫는태그를 동시에 수정가능하게 함
Auto Close Tag : <div>만 입력을 해도 </div>가 자동으로 작성이 되게 함

이 두개 추천드림니당